tests/char_exposure_test.cpp

```diff
@@ -0,0 +1,123 @@
+#include "catch/catch.hpp"
+
+#include "avatar.h"
+#include "player_helpers.h"
+
+// Tests for Character bodypart exposure
+//
+// Covers functions:
+// - Character::bodypart_exposure
+
+TEST_CASE( "character body part exposure", "[char][bodypart][exposure]" )
+{
+    Character &dummy = get_player_character();
+    clear_avatar();
+
+    bodypart_id head = bodypart_str_id( "head" );
+    bodypart_id torso = bodypart_str_id( "torso" );
+    bodypart_id arm_l = bodypart_str_id( "arm_l" );
+    bodypart_id arm_r = bodypart_str_id( "arm_r" );
+    bodypart_id hand_l = bodypart_str_id( "hand_l" );
+    bodypart_id hand_r = bodypart_str_id( "hand_r" );
+    bodypart_id leg_l = bodypart_str_id( "leg_l" );
+    bodypart_id leg_r = bodypart_str_id( "leg_r" );
+    bodypart_id foot_l = bodypart_str_id( "foot_l" );
+    bodypart_id foot_r = bodypart_str_id( "foot_r" );
+
+    std::map<bodypart_id, float> bp_exposure;
+
+    GIVEN( "nothing is worn" ) {
+        dummy.worn.clear();
+
+        THEN( "exposure is 100%% on all body parts" ) {
```

Comment:
Should the `%` characters be escaped in this context?  I wouldn't have thought so.

Reply:
You are right, looks like it is not necessary. I had done the same for several previous test cases, but forget why - could be just Vim's syntax highlighting steered me wrong. Will adjust.